sparkmagic/sparkmagic/tests/test_handlers.py

```diff
@@ -62,7 +64,7 @@ def setUp(self):
 
         # Mock request
         self.request = MagicMock()
-        self.request.body = json.dumps({"path": self.path, "username": self.username, "password": self.password, "endpoint": self.endpoint})
```

Comment:
Please add two tests where you don't send the `auth` parameter at all and still get no auth and basic auth as values for auth.